test/unit/routers/alpha-router/alpha-router.test.ts

```diff
@@ -1850,44 +1944,39 @@ describe.only('alpha router', () => {
       sinon.assert.notCalled(mockOnChainQuoteProvider.getQuotesManyExactOut);
     });
 
-    test('succeeds to route and generates calldata on v3 only and simulates', async () => {
+    test('succeeds to route and generates calldata on v2 only', async () => {
```

Comment:
Whys this and the below unit tests changed?

Reply:
I changed the order of the specs

Reply:
I put the simulation specs at the end of each describe to organize better. Also I removed some unnecessary params from the simulation specs.

Reply:
also, i added more specs in simulation-provider.test.ts, so test coverage actually increased.

Reply:
i can change the order of the tests back if you think it should not change, just lmk